src/Aspire.Dashboard/Components/Dialogs/SettingsDialog.razor

```diff
@@ -15,7 +17,15 @@
             <FluentRadio Value="@ThemeManager.ThemeSettingDark">@Loc[nameof(Dialogs.SettingsDialogDarkTheme)]</FluentRadio>
         </FluentRadioGroup>
 
+        <FluentCombobox TOption="CultureInfo"
```

Comment:
I think this should be a FluentSelect. There is a known list of values, and it's not that large, so typing in a combo box doesn't seem useful.

Reply:
changed